packages/openapi-ts/global.d.ts

```diff
@@ -0,0 +1,15 @@
+// global.d.ts
```

Comment:
Is this file for the utils/runtime module?

Reply:
Yes, if not TypeScript (node) thinks it is an undeclared type since it is not aware of Deno and Bun (rightfully so)